internal/models/plugin.go

```diff
@@ -5,6 +5,8 @@ import (
 	"encoding/json"
 )
 
+type PluginFn func() Plugin
```

Comment:
Maybe PluginCreationFn ? To be more specific ?

Reply:
Hm, yeah, will change it. Or maybe `PluginConstructor` or `PluginConstructorFn`? 💡